src/main/resources/resources/logisim/strings/gui/gui.properties

```diff
@@ -608,61 +608,61 @@ windowTitle = Window
 windowToolbarHidden = Hidden
 windowToolbarImportant = Important: changing the below values may have unpredictable results!
 windowToolbarLocation = Toolbar location:
-windowToolbarLookandfeel = Look and Feel:
-windowToolbarPleaserestart = Please restart logisim.
+windowToolbarLookandfeel = Look and feel:
+windowToolbarPleaserestart = Please restart Logisim.
 windowToolbarPreview = Preview
 windowToolbarReset = Reset window layout to Logisim's defaults
 windowToolbarZoomfactor = Zoom factor
-windowCanvasBgColor = Canvas Background Color:
-windowComponentColor = Component Color:
-windowGridBgColor = Grid Background Color:
-windowGridDotColor = Grid Dot Color:
-windowGridZoomedDotColor = Grid Zoomed Dot Color:
+windowCanvasBgColor = Canvas background color:
+windowComponentColor = Component color:
+windowGridBgColor = Grid background color:
+windowGridDotColor = Grid dot color:
+windowGridZoomedDotColor = Grid zoomed dot color:
 windowGridColorsReset = Reset grid colors to Logisim's defaults
 windowSetAutoScaleFactor = Set zoom factor to Logisim's default
 windowCanvasLocation = Main canvas location:
 #
 # prefs/HotkeyOptions.java
 #
-hotkeyOptMenuKeyHeader = Menu Shortcuts That Require the %s Key:
-hotkeyOptNormalKeyHeader = Context Dependent Hotkeys That Do Not Require the %s Key:
-hotkeyOptOrientDesc = Component's Orientation Shortcuts
+hotkeyOptMenuKeyHeader = Menu shortcuts that require the %s key:
+hotkeyOptNormalKeyHeader = Context dependent hotkeys that do not require the %s key:
+hotkeyOptOrientDesc = Component's orientation shortcuts
 hotkeyOptHelp = To set the key bindings, click the bordered area right-side and press your expected keys in the keypad. Click the check mark to save, or discard by clicking the cross mark.
-hotkeyOptTitle = Hotkey Settings
-hotkeyOptResetBtn = Reset to Default
+hotkeyOptTitle = Hotkey settings
+hotkeyOptResetBtn = Reset to defaults
 hotkeySimAutoPropagate = Auto-Propagate
-hotkeySimReset = Reset Simulation
-hotkeySimStep = Single-Step Propagation
-hotkeySimTickHalf = Manual Tick Half Cycle
-hotkeySimTickFull = Manual Tick Full Cycle
-hotkeySimTickEnabled = Auto-Tick Enabled
-hotkeyEditUndo = Edit Undo
-hotkeyEditRedo = Edit Redo
-hotkeyWindowClose = Close the Window
-hotkeyWindowMinimize = Minimize the Window
-hotkeyFileExport = Export File
-hotkeyFilePrint = Print File
-hotkeyErrMeta = Must contain the %s key.
-hotkeyErrConflict = This key conflicts with: %s.
-hotkeyDirNorth = North
-hotkeyDirSouth = South
-hotkeyDirEast = East
-hotkeyDirWest = West
+hotkeySimReset = Reset simulation
+hotkeySimStep = Single-step propagation
+hotkeySimTickHalf = Manual tick half cycle
+hotkeySimTickFull = Manual tick full cycle
```

Comment:
Shouldn't it be "Manually tick..."?

@davidhutchens: Could you maybe comment on this?

Reply:
Now that you point it out, it does seem slightly off. I would actually delete "Manual" and just write "Tick half cycle" and "Tick full cycle". Manual(ly) provides no additional information.

But note that "Manual Tick Half Cycle" is the name of the menu item. So either way, we should be consistent. I might just leave it be.